refs/high-level-conceptualization/README.md

```diff
@@ -1,111 +1,122 @@
 
-_This high-level architecture describes the conceptualization behind blidnet’s software, in terms of the information it deals with and the reality it enables.
 This conceptualization does not directly translate to any database structure, workflow, or code. It aims to inform software designers’ thinking, guide design choices, and reduce confusion._
 
-## THE MAIN CONCEPT - DATA CAPTURE
+## TERMINOLOGY
+
+- The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119)
+- The word "CAN" denotes ability of someone or something, and is interpreted as "MUST be able to"
+- All terms defined in [RFC-Lexicon-2](../lexicon/RFC-Lexicon-2.md) are to be interpreted as described there
+
+### THE MAIN CONCEPT - DATA CAPTURE
 
 <img width="900" alt="organisation" src="./img/organization.png">
 
-A Data Capture is the most central concept of interest in blindnet.
+A [Data Capture](../lexicon/RFC-Lexicon-2.md#data-capture) is the most central concept of interest in [blindnet devkit](../lexicon/RFC-Lexicon-2.md#blindnet-devkit).
 
-A Data Capture is a compound unit (composed of other units) of confidential information provided by a Submitter to an Organisation.
 It corresponds to a particular data exchange need, and can be composed of files and other data fields of different types.
-A Data Capture can be captured at once or over the course of multiple interactions. One or more Submitters can complete a Data Capture.
+A Data Capture can be captured at once or over the course of multiple interactions. One or more [Submitters](../lexicon/RFC-Lexicon-2.md#submitter) can complete a Data Capture.
 
 Examples of a need include: medical history of a new patient (doctor); candidate CV (company hiring); a set of legal proofs in a lawsuit provided by a client (lawyer); identity documents of founders in the investment KYC process (VC fund).
 
-## FORM FOLLOWS FUNCTION
+### FORM FOLLOWS FUNCTION
 
-By a Privateform we understand the interface, the data structures and to them associated settings and logic allowing to capture Data Captures.
+A [Privateform](../lexicon/RFC-Lexicon-2.md#privateform) includes the interface, the data structures and to them associated settings and logic allowing to capture [Data Captures](../lexicon/RFC-Lexicon-2.md#data-capture).
 
-Settings may include configurations related to delays of conservation and expiry of Data Captures, confidentiality and access rights to captured data, consent collection options, view/decrypt rights and similar.
+Settings MAY include configurations related to delays of conservation and expiry of Data Captures, confidentiality and access rights to captured data, Consent collection options, view/decrypt rights and similar.
```

Comment:
Consider defining `Consent` in a lexicon

Reply:
Since Consent is a Legal Base, we'd then also have to include that term in the Lexicon. 
It is potentially a good idea to include all terms defines in any document, in the lexicon.
However it is not the idea with which I wrote the current lexicon, but I rather put in its scope the terms that I believe will have large implications for a multitude of other documents.

So I think this can be added later, if proven useful.